app/components/utils/mandatee-selector.js

```diff
@@ -67,4 +72,31 @@ export default class MandateeSelector extends Component {
       this.mandateeOptions = this.loadMandatees.perform();
     }
   }
+
+  async loadCurrentBodyMandatees(searchTerm) {
```

Comment:
@TaneeG please add a comment here along the lines of: 

```
/* WARNING: use of the :has-no:  filter is officially only documented for use with relations, not properties. However, at the time of writing, this method works in the case of xsd:datetime as well. A test case will be made to check if this keeps working. If this test case should ever fail, the loadCurrentBodyMandatees method can be adjusted to fetch the mandatee with the most recent start date, and then use that start date to get the current government body. This is less efficient than the current method, but would require no hard-coding and does not rely on :has-no: */
```

@ValenberghsSven in the next sprint, could a test case please be added to check whether the `:has-no:` filter keeps working for xsd:datetimes? Created KAS-3402 for this. Thanks!